src/schema/validator.js

```diff
@@ -356,6 +444,23 @@ export class SchemaValidator {
         const manifestVersion =
           (rootData && rootData.manifest_version) || MANIFEST_VERSION_DEFAULT;
         const res = condFn(keywordSchemaValue, manifestVersion);
+
+        // If the min/max_manifest_version is set on a schema entry of type array,
+        // propagate the same keyword to the `items` schema, which is needed to
+        // - be able to recognize that those schema entries are also only allowed on
+        //   certain manifest versions (which becomes part of the linting messages)
+        // - be able to filter out the validation errors related to future (not yet
+        //   supported) manifest versions if they are related to those schema entries
+        //   (which happens based on the current or parent schema in the `filterErrors`
+        //   helper method).
```

Comment:
Would be nice to file an issue for this + add `TODO: <url of the issue>` here.

Reply:
:+1: totally, I'm going to file an issue upfront and mention it here in this comment